lib/cli_common/cli_common/phabricator.py

```diff
@@ -243,6 +243,55 @@ def load_or_create_build_autotarget(self, object_phid, target_keys):
         )
         return res['targetMap']
 
+    def search_buildable(self, object_phid):
+        '''
+        Search HarborMaster buildables linked to an object (diff, revision, ...)
+        '''
+        constraints = {
+            'objectPHIDs': [object_phid, ],
+        }
+        out = self.request(
+            'harbormaster.buildable.search',
+            constraints=constraints,
+        )
+        return out['data']
+
+    def search_build(self, buildable_phid, plans=[]):
+        '''
+        Search HarborMaster build for a buildable
+        Supports HarborMaster Buidl Plan filtering
+        '''
+        constraints = {
+            'buildables': [buildable_phid, ],
+            'plans': plans,
+        }
+        out = self.request(
+            'harbormaster.build.search',
+            constraints=constraints,
+        )
+        return out['data']
+
+    def find_diff_build(self, diff_phid, build_plan_phid):
+        '''
+        Find a specific build for a Diff and an HarborMaster build plan
+        '''
+        assert diff_phid.startswith('PHID-DIFF-')
```

Comment:
This could also be a revision PHID (e.g. in ccov we'll probably use that to find the buildable